.github/workflows/lint.yml

```diff
@@ -0,0 +1,17 @@
+name: Run linters
+
+on: [push]
+
+jobs:
+  golang-ci-lint:
+    name: GolangCI
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v1
+      - name: Run GolangCI-Lint
+        uses: docker://matousdz/golangci-lint-action:v1.0.0
```

Comment:
Can we have it in `golangci` namespace?
It would be great to have something like
```
- name: Run GolangCI-Lint Action by building from repository
  uses: golangci/golangci-lint-action@v1
```

Probably we can ask @jirfag to create a repo for that action.

Reply:
I agree, please, change to `golangci`